testing/jormungandr-testing-utils/src/testing/fragments/sender.rs

```diff
@@ -142,6 +142,21 @@ impl<'a, S: SyncNode + Send> FragmentSender<'a, S> {
         self.send_fragment(from, fragment, via)
     }
 
+    pub fn send_transaction_with_validity<A: FragmentNode + SyncNode + Sized + Send>(
```

Comment:
This sounds like validity is an optional field, and indeed we have also `send_transaction`. I would pick another less ambiguous name, and also understand why we have these two methods. If `send_transaction` is a convenience method to do some date magic it could just pick the value and reuse `send_transaction_with_validity`

Reply:
Oh my bad, that's leftover code from an attempt to clean up the testing API. I got rid of it. `BlockDate` magic aside, the validity for a transaction is defined upon construction of the `FragmentSender` and all transactions created by it use that `BlockDate`.